.github/workflows/test-operator-wandb.yaml

```
@@ -0,0 +1,71 @@
+  test:
+    name: Test Chart
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s-version: ["v1.32.1", "v1.31.4", "v1.30.8"]
+        configuration: ["default", "separate-pods", "runs-v2-bufstream"]
+    runs-on: ubuntu-latest
+    environment: Helm Charts
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)